### DISCUSSION TITLE
Support "Hit Highlighted Properties" in PnP Search Result Webpart when use SharePoint Search data source

### DISCUSSION THREAD

**Discussion by wuxiaojun514** [microsoft-search/pnp-modern-search#1688] [Ideas]
**version: 4.5.3**
I cannot find **"Hit Highlighted Properties"** in the property panel of PnP Search Result Webpart.
This is quite useful property when we call sharepoint search rest api, it let user know which property hit the search query.
![image](https://user-images.githubusercontent.com/29603048/147643898-7fe5a1ae-ee1c-43b6-9635-e07797375428.png)
We use it heavily in class search, so we want PnP Search can also support it

**Comment by wobba** [comment]
Manually enter it to the list of properties you want returned and knock yourself out :)

![image](https://user-images.githubusercontent.com/51104/147949990-373f4f9b-e9de-4189-a010-7d05fa6d503d.png)

**Comment by wuxiaojun514** [comment]
@wobba 

The "HitHighlightedProperties" in selected properties will be empty if you don't add fields in hithighlightedproperties parameter when call sharepoint search rest api
![image](https://user-images.githubusercontent.com/29603048/148040233-eb70b4fa-aa5a-4d49-a53f-df2437d0e2d5.png)

I added fields in "HitHighlightedProperties" parameter, then I can get data in the result
![image](https://user-images.githubusercontent.com/29603048/148039851-896250ee-eba5-410d-abe3-d6ce5982c3b4.png)

So I hope we can set **HitHighlightedProperties** parameter in Property Panel of PnP Search Result Webpart

**Reply by wobba** [reply]
Ahh.. got it. Added it as an idea for someone to address.
